Rakefile

```diff
@@ -32,7 +32,7 @@ task make_no_debug: [:templates] do
 end
 
 # decorate the gem build task with prerequisites
-task build: [:templates, :check_annotations, :check_manifest]
+task build: [:check_manifest, :templates, :check_annotations]
```

Comment:
I changed the order so `:check_manifest` is done first and fails quickly & early, before compiling stuff, which would make the error less obvious.